packages/web-twig/src/Resources/components/Modal/stories/ModalDefault.twig

```diff
@@ -97,46 +97,52 @@
     </ModalDialog>
 </Modal>
 
-<Button data-spirit-toggle="modal" data-spirit-target="#example-custom-height">
-    Open Modal with Custom Height
+<Button data-spirit-toggle="modal" data-spirit-target="#example-dropdown">
+    Open Modal with Dropdown
 </Button>
 
-<Modal id="example-custom-height" titleId="example-custom-height-title">
-    <ModalDialog
-        elementType="form"
-        isExpandedOnMobile={ false }
-        maxHeightFromTabletUp="700px"
-        method="dialog"
-        preferredHeightOnMobile="400px"
-        preferredHeightFromTabletUp="500px"
-    >
-        <ModalHeader modalId="example-custom-height" titleId="example-custom-height-title">
-            Modal with Custom Height
+<Modal id="example-dropdown" titleId="example-dropdown-title">
+    <ModalDialog>
+        <ModalHeader modalId="example-dropdown" titleId="example-dropdown-title">
+            Modal Title
         </ModalHeader>
         <ModalBody>
 
             <!-- Content: start -->
-            <p class="d-tablet-none">
-                This modal has a custom height of <code>400px</code>.
-                <br /><br />
-                The max height cannot be customized on mobile though.
-            </p>
-            <p class="d-none d-tablet-block">
-                This modal has a custom height of <code>500px</code>.
-                <br /><br />
-                The max height of this modal is <code>700px</code>.
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquam at excepturi laudantium magnam mollitia
+                perferendis reprehenderit, voluptate. Cum delectus dicta ducimus eligendi excepturi natus perferendis
+                provident unde. Eveniet, iste, molestiae?
             </p>
+            <DropdownWrapper>
```

Comment:
We are renaming the Dropdown in another PR. Please, do not forget to change it also here.

Reply:
Done.